sci2zero/TeslaRIS-frontend
# Feature/identifier validation

## Description

Added validations for all major identifiers. Improved error nadling and notificaton system.

## How should this be tested?

When creating/updating any entity, try to supply wrong identifier format, you will not be able to submit the form. Try to input a identifier that is allready in use, you should get an error message indicating that it is allready in use.

## Checklist
